qiskit_experiments/experiment_data.py

```diff
@@ -64,23 +41,13 @@ def __init__(
         Raises:
             ExperimentError: If an input argument is invalid.
         """
-        # Experiment class object
         self._experiment = experiment
-        self._metadata = experiment._metadata() if experiment else {}
-
-        # Terra ExperimentDataV1 attributes
-        self._backend = backend
-        self._id = str(uuid.uuid4())
-        if experiment is not None:
-            self._type = experiment._type
-        else:
-            self._type = None
-        job_ids = job_ids or []
-        self._jobs = OrderedDict((k, None) for k in job_ids)
-        self._data = []
-        self._figures = OrderedDict()
-        self._figure_names = []
-        self._analysis_results = []
+        super().__init__(
+            experiment_type=experiment.experiment_type if experiment else "unknown",
```

Comment:
Can we say just `None` rather than `unknown` as in the original implementation?

Reply:
I moved the `unknown` default to `DbExperimentDataV1` in 0d6e3c9, since it's a required field in results db.